src/conn/DBConnection.h

```diff
@@ -30,10 +30,10 @@ class DBConnection
     
     private:
         //Connection Variables
-        sql::Connection *conn;
-        sql::PreparedStatement *prep_stmt;
+        sql::Connection *_conn;
+        sql::PreparedStatement *_prep_stmt;
```

Comment:
I agree. I am happy to change them.
One question though. Will the `_connection` variable for this class clash with the `_connection` variable from the model class, or are they protected by scope?